Scala2-Functional-Programming/Week6/forcomp/src/main/scala/forcomp/Anagrams.scala

```diff
@@ -33,10 +35,10 @@ object Anagrams extends AnagramsInterface {
    *
    *  Note: you must use `groupBy` to implement this method!
    */
-  def wordOccurrences(w: Word): Occurrences = ???
+  def wordOccurrences(w: Word): Occurrences = w.groupBy(c=>c.toLower).map(x => (x._1, x._2.length)).toList.sorted
```

Comment:
Generally, we don't like seeing `_1` and `_2` in the code. it makes reading more difficult.
Specially here, there is a simpler solution.
Try using `mapValues` func

Reply:
Done